XMonad/Doc/Extending.hs

```diff
@@ -177,7 +170,7 @@ In the @XMonad.Layout@ namespace you can find modules exporting
 contributed layout algorithms, such as a tabbed layout, a circle, a spiral,
 three columns, and so on.
 
-You will also find modules which provide facilities for combining
```

Comment:
Seems to me that "which" and "that" are equally applicable here?

Reply:
Indeed; I remember my English teacher telling me not to use which in integrated relative clauses but apparently that was just wrong.  I'll probably change this back then to reduce diff noise

Reply:
English teachers teach a lot of guff derived from trying to turn English into Latin.